BunnyHopMaster/Assets/Scripts/Portal/CameraMove.cs

```diff
@@ -25,36 +19,31 @@ private void Awake()
         playerCamera = Camera.main;
     }
 
+    private void Start()
+    {
+        sensitivityMultiplier = OptionsPreferencesManager.GetSensitivity();
+    }
+
     private void Update()
     {
+        if(Time.timeScale == 0)
+        {
+            return;
+        }
+
         // Rotate the camera.
         var rotation = new Vector2(-Input.GetAxis("Mouse Y"), Input.GetAxis("Mouse X"));
-        var targetEuler = TargetRotation.eulerAngles + (Vector3)rotation * cameraSpeed;
+        var targetEuler = TargetRotation.eulerAngles + (Vector3)rotation * sensitivityMultiplier;
         if(targetEuler.x > 180.0f)
         {
             targetEuler.x -= 360.0f;
         }
         targetEuler.x = Mathf.Clamp(targetEuler.x, -75.0f, 75.0f);
         TargetRotation = Quaternion.Euler(targetEuler);
 
-        Quaternion rotationToApply = Quaternion.Slerp(playerCamera.transform.rotation, TargetRotation,
-            Time.deltaTime * 15.0f);
+        Quaternion rotationToApply = TargetRotation;

```

Comment:
Where does this come from!?!?!? `TargetRotation`

Reply:
AHHH nice find